jina/orchestrate/deployments/__init__.py

```diff
@@ -282,6 +306,13 @@ def update_pod_args(self):
         else:
             self.pod_args = self._parse_args(self.args)
 
+        if self.external:
```

Comment:
What does this do?

Reply:
When the deployment is not external then we do not expect the user to pass ports and hosts. Therefore we should not re-assign the port and host already associated with a given pod